lib/contentful/client.rb

```diff
@@ -296,6 +298,16 @@ def sync(options = { initial: true })
 
     private
 
+    def ensure_select_is_doable(query)
+      return unless query.key?(:select)
+
+      query[:select] = query[:select].split(',').map(&:strip) if query[:select].is_a? String
+
+      query[:select] = query[:select].reject { |p| p.start_with?('sys.') }
+
```

Comment:
Super minor code style thing but I feel like the extra line breaks between each line in this method are distracting and not serving a structural purpose.